CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+
+If you want to go the usual route and run the project locally, though:
+
+- [Install Node.js](https://nodejs.org/en/download/)
+- [Fork the project](https://guides.github.com/activities/forking/#fork)
+
+Then in your terminal:
+
+- `cd path/to/your/clone/examples/<example_directory>`
+- `npm install`
+- `npm dev`
+- `mage build:backend` (if the plugin has a backend)
+- `docker compose up`
+
+And you should be ready to go!
+
+> [!NOTE]
+> This repository is a collection of Grafana plugins with their own dependency lock files, dev environments, etc. That is to say it is not a monorepo in the traditional sense. To develop against a plugin you must navigate inside it's directory and run the above commands for each example you would like to contribute to.
+
+## Contributor License Agreement (CLA)
+
+Before we can accept your pull request, you need to [sign our CLA](https://grafana.com/docs/grafana/latest/developers/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
+
+## Contribute Documentation
+
+Documentation is a super important, critical part of this project. Docs are how we keep track of what we're doing, how, and why. It's how we stay on the same page about our policies. And it's how we tell others everything they need in order to be able to use this project -- or contribute to it. So thank you in advance.
+
+Documentation contributions of any size are welcome! Feel free to file a PR even if you're just rewording a sentence to be more clear, or fixing a spelling mistake!
+
+To contribute documentation:
+
+- [Set up the project](#project-setup).
+- Edit or add any relevant documentation.
+- Make sure your changes are formatted correctly and consistently with the rest of the documentation.
+- Re-read what you wrote, and run a spellchecker on it to make sure you didn't miss anything.
+- Go to https://github.com/grafana/grafana-plugin-examples/pulls and open a new pull request with your changes.
+- If your PR is connected to an open issue, add a line in your PR's description that says `Fixes: #123`, where `#123` is the number of the issue you're fixing.
+
+Once you've filed the PR:
+
+- One or more maintainers will use GitHub's review feature to review your PR.
+- If the maintainer asks for any changes, edit your changes, push, and ask for another review.
+- If the maintainer decides to pass on your PR, they will thank you for the contribution and explain why they won't be accepting the changes. That's ok! We still really appreciate you taking the time to do it, and we don't take that lightly. 💚
+- If your PR gets accepted, it will be marked as such, and merged into the `latest` branch soon after. Your contribution will be distributed to the masses once the PR is merged.
```

Comment:
Should this be `main` branch?

```suggestion
- If your PR gets accepted, it will be marked as such, and merged into the `main` branch soon after. Your contribution will be distributed to the masses once the PR is merged.
```